.github/workflows/release-please.yml

```diff
@@ -0,0 +1,63 @@
+name: Release Please
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-22.04
+
+    outputs:
+      dotnet-sdk-internal-released: ${{ steps.release.outputs.release_created }}
+      dotnet-sdk-internal-tag_name: ${{ steps.release.outputs.tag_name }}
+
+    steps:
+      - uses: googleapis/release-please-action@v4
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          target-branch: ${{ github.ref_name }}
+
+  release-sdk-internal:
+    runs-on: ubuntu-22.04
+    needs: release-please
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
```

Comment:
You should add `issues: write` as well. GH broke permissions recently, and PR: write isn't enough for RP anymore.